src/config/webpack.common.js

```diff
@@ -17,16 +17,22 @@ const copies = {
 const { eslintConfig, main: entryPoint } = require(resolveExtensionPath("package.json"));
 const eslintEnabled = eslintConfig || fs.readdirSync(extensionPath).find(f => f.startsWith(".eslintrc"));
 const jsLoaders = [{
-    loader: require.resolve("babel-loader"),
+    loader: "babel-loader",
     options: {
         presets: [
-            [require.resolve("@babel/preset-env"), {
-                targets: {
-                    chrome: 45,
-                    safari: "9.1",
-                    firefox: 45
+            [
+                "@babel/preset-env",
+                {
+                    useBuiltIns: "usage",
+                    corejs: 3,
```

Comment:
I think we should add `core-js@3` to the list of "dependencies" of zem since this is going to be needed in case a feature that requires polyfill is used.